website/config/routes.js

```diff
@@ -278,6 +278,7 @@ module.exports.routes = {
   'GET /blog':                   '/articles',
   'GET /brand':                  '/logos',
   'GET /get-started':            '/try-fleet/register',
+  'GET /docs/using-fleet/fleet-ui#configuring-agent-options': '/docs/using-fleet/configuration-files#agent-options',
```

Comment:
@eashaw I added a redirect here because we link users to `/fleet-ui#configuring-agent-options` in the Fleet UI. 

These docs changes will break this link for anyone using Fleet 4.19.1 or lower.

Is this the best way to handle this broken link?

Reply:
@noahtalerman This redirect would **not** work. In-page anchor links (e.g., `#configuring-agent-options`) aren't sent to the server, so we wouldn't be able to handle them with a redirect in `routes.js`. We would probably have to handle this with a client-side redirect on the documentation's page script.

Reply:
> This redirect would not work.

@eashaw thanks for the heads up.

> We would probably have to handle this with a client-side redirect on the documentation's page script.

Got it. I don't think moving this specific section is valuable enough to put the effort in to make this change.

I will bring the old section back in so that we can move quickly.